cds/config.py

```diff
@@ -187,6 +187,13 @@ def _(x):
 }
 # Default Elasticsearch document type.
 SEARCH_DOC_TYPE_DEFAULT = None
+
+# Legacy host Elasticsearch
+LEGACY_STATS_ELASTIC_HOST = '127.0.0.1'
+# Legacy port Elasticsearch
+LEGACY_STATS_ELASTIC_PORT = 9200
```

Comment:
Port was 9199, is it correct?
I mean, it is overridden when deployed, just wondering...